### DISCUSSION TITLE
关于release数量显示不准的问题

### DISCUSSION THREAD

**Discussion by eko5624** [zhongfly/mpv-build#34] [General]
比如我这个仓库 https://github.com/eko5624/mpv-win64
右侧Releases数目显示的20，但实际上只有3个。因为我把之前的release删了一些，只留了这3个。但点Releases标签依然显示了2页，不过第2页是空白的。应该怎么让Releases标签显示的数目和实际数目相匹配呢？

**Comment by zhongfly** [comment]
我觉得你应该找github客服，反馈一下bug…这种问题我不了解
